### ISSUE TITLE
Complete instructions to get started?

### ISSUE THREAD

**Issue by dncohen** [fragmenta/fragmenta-cms#19]
I'm following the instructions on http://fragmenta.eu/develop, but it feels like its missing important details that a first-time user like myself doesn't know.  I'm new to golang as well, which could be part of the problem.

Following the instructions as-is gets this:

~~~
[user@sandbox fragmenta-test]$ fragmenta server
01:21:58 
------
Fragmenta version: 1.3.3
------
01:21:58 Building server...
01:21:58 Running go fmt at ./...
01:21:59 Building server at /home/user/gocode/src/fragmenta-test/bin/fragmenta-server-local
01:22:00 Build completed successfully in 1.700707279s
01:22:00 Launching server...
2016/03/21 01:22:00 #info Opened log file at log/development.log
2016/03/21 01:22:00 #info Finished loading assets in 76.334µs
2016/03/21 01:22:00 #info Finished loading templates in 5.313037ms
2016/03/21 01:22:00 Error reading database pq: Ident authentication failed for user "fragmenta-test_server"
~~~

The last error gave me enough of a clue to find secrets/fragmenta.json.  Here I see answers to questions like what database and database user I'll have to make, and what port the server will listen on.  But there's 3 sets of answers (development, production, test) and I'm not even sure which of those environments is running!

**Comment by kennygrant** [comment]
I agree it would be useful to give instructions on setup, and a clearer explanation of the config file, thanks for letting me know it is unclear.

**Comment by umamiMike** [comment]
I am currently getting the same results as  dncohen.  I am assuming I have to make a user in postgres that matches?  honestly I am not exactly sure. I am working through it but would love a leg up.  The project looks promising.  One thing I will post as an issue once I am able to fully install/use/ contribute is it currently doesn't seem to be able to handle multiple gopath entries.  Thanks again for any illumination somebody can throw on the db connection.

**Comment by Hasnaa-Ibraheem** [comment]
Hi UmamiMike,

I believe this is a late answer to your question, but in case any other developer reached here using search engine.
You need to check your secrets-> fragmenta.json file, inside it make sure that you have the right configurations: db, db_adapter, db_user, db_pass
"db" stands for your database name, so if it has value like mycms_development you have to have a database named: mycms_development
"db_adapter": defines which database driver shall be used
"db_user"/"db_pass" for the username and password for the database specified in the "db" field.

I hope this helps.

**Comment by humhann** [comment]
I managed to get it working on macOS, hope it helps somebody: https://medium.com/@humhann/getting-started-with-fragmenta-cms-on-macos-a54549aaacb2
